libindy/tests/utils/pool.rs

```diff
@@ -26,15 +26,16 @@ struct PoolConfig {
 pub fn create_genesis_txn_file(pool_name: &str,
                                txn_file_data: &str,
                                txn_file_path: Option<&Path>) -> PathBuf {
-    let txn_file_path = txn_file_path.map_or(
-        environment::tmp_file_path(format!("{}.txn", pool_name).as_str()),
-        |path| path.to_path_buf());
-
-    if !txn_file_path.parent().unwrap().exists() {
-        fs::DirBuilder::new()
-            .recursive(true)
-            .create(txn_file_path.parent().unwrap()).unwrap();
-    }
+    let txn_file_path= match txn_file_path {
```

Comment:
Seems like behavior is changed now for case of specified path and non-existing parent directory.
But it should not be a problem for this test helpers